### PR TITLE
[ADD] New module <mrp_production_note>

### PR DESCRIPTION
**Pull request by oihane**

### CODE REVIEW

File: mrp_production_note/README.rst [hunk 1]
```diff
@@ -0,0 +1,42 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Notes in production orders
+==========================
+
+This module creates in production orders a new field to add notes
```
Comment (pedrobaeza) [inline mrp_production_note/README.rst]: You can add at the end `in rich text format`

**Comment by pedrobaeza** [comment]
:+1:

**Comment by oihane** [comment]
Changed

**Comment by mathi123** [comment]
:+1:

**Comment by bwrsandman** [comment]
:+1: 
Simple enough but is the `####` wall needed?

**Comment by bwrsandman** [comment]
Someone should do a functional review using runbot

**Comment by anajuaristi** [comment]
The module is being using in production in several customers so it should be functionally validated by now. 

:+1:

**Comment by mathi123** [comment]
I also did a functional review locally.

**Pull request merged**